### PR TITLE
style: updates Pine component colors and type styles

### PR DESCRIPTION
**Pull request by anechol**
# Description

Updates Pine components color and type styles. This **does not** address component design refreshes/changes/updates. Only component that have updated [Figma](https://www.figma.com/design/CC1YmaGKHnsvB28yLY9mEH/%F0%9F%A7%A9-%5BWIP-DO-NOT-USE%5D-Mercury-components?node-id=351-5688&t=jmmdJIaWJ6301VRm-1) files were addressed.

There are also some instances were component level tokens were added, updated, renamed, or removed.

[DSS-1031](https://kajabi.atlassian.net/browse/DSS-1031)

## Type of change

Please delete options that are not relevant.
If your type of change is not present, add that option.

- [x] Styles (adds, removes, or updates component styles)

# How Has This Been Tested?

Please describe the tests you've added and run to verify your changes.
Provide instructions so that we can reproduce.
Please also list any relevant details for your test configuration.

- [ ] unit tests
- [ ] e2e tests
- [ ] accessibility tests
- [x] tested manually
- [ ] other:

**Test Configuration**:

- Pine versions:
- OS:
- Browsers:
- Screen readers:
- Misc:

# Checklist:

If not applicable, leave options unchecked.

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing tests pass locally with my changes
- [ ] Design has QA'ed and approved this PR

[DSS-1031]: https://kajabi.atlassian.net/browse/DSS-1031?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: libs/core/src/components/pds-accordion/pds-accordion.scss [hunk 1]
```diff
@@ -23,7 +23,7 @@ details {
   --spacing-summary-padding-inline-start: var(--pine-spacing-100);
   --spacing-summary-padding-inline-end: var(--pine-spacing-050);
 
-  --typography-default: var(--pine-font-weight-medium) var(--pine-font-size-100)/var(--pine-line-height-150) var(--pine-font-family-inter);
+  --typography-default: var(--pine-typography-body-md-medium);
```
Comment (ju-Skinner) [inline libs/core/src/components/pds-accordion/pds-accordion.scss:26]: 📓 For later discussion

Could we rename this? This seems redundant.

--[DS name]-[category]-[target]-[font-weight]. I can explain at a later time

File: libs/core/src/components/_internal/pds-label/pds-label.scss [hunk 2]
```diff
@@ -1,19 +1,14 @@
...
 
   color: var(--color-text-default);
-  font-family: var(--font-family-default);
-  font-weight: var(--font-weight-default);
-  letter-spacing: var(--letter-spacing-default);
+
+  [disabled] &, *:disabled & {
+    color: var(--color-text-disabled);
+  }
 }
```
Comment (pixelflips) [inline libs/core/src/components/_internal/pds-label/pds-label.scss]: nitpick, missing new line. checkout https://marketplace.visualstudio.com/items?itemName=jdforsythe.add-new-line-to-files if you're using vscode.

**Review by pixelflips — APPROVED** [review]

File: libs/core/src/components/_internal/pds-label/pds-label.scss (hunk 2 above)
Comment (anechol) [reply libs/core/src/components/_internal/pds-label/pds-label.scss]: Thanks for this, installing now!

File: libs/core/src/components/pds-loader/pds-loader.tsx [hunk 3]
```diff
@@ -69,20 +69,20 @@ export class PdsLoader {
...
+                  <stop offset="100%" stop-opacity="0.5" stop-color="var(--pine-color-mercury-400)" />
                 </linearGradient>
               </defs>
 
               <g class="pds-loader__spinner-path">
                 <path stroke="url(#spinner-secondHalf)" d="M 4 100 A 96 96 0 0 1 196 100" />
                 <path stroke="url(#spinner-firstHalf)" d="M 196 100 A 96 96 0 0 1 4 100" />
 
-                <path stroke="currentColor" stroke-linecap="round" d="M 4 100 A 96 96 0 0 1 4 98" />
+                <path stroke="var(--pine-color-mercury-400)" stroke-linecap="round" d="M 4 100 A 96 96 0 0 1 4 98" />
```
Comment (ju-Skinner) [inline libs/core/src/components/pds-loader/pds-loader.tsx]: As I was rebasing this morning, I noticed this. How do we allow the customer to change the color if we set these values? The purpose of using `currentColor` was to allow a color to be set via the component. Let's discuss this before merging.

File: libs/core/.storybook/preview.js [hunk 4]
```diff
@@ -16,6 +16,13 @@ if (docsJson) { setStencilDocJson(docsJson) }
 const preview = {
   parameters: {
     actions: { argTypesRegex: "^on[A-Z].*" },
+    options: {
+      storySort: {
+        method: 'alphabetical',
+        order: ['Foundations', 'Guides', 'Components', 'Resources'],
+        locales: 'en-US',
+      },
+    }
```
Comment (ju-Skinner) [inline libs/core/.storybook/preview.js:25]: For knowledge sharing, this reorders the navigation.

**Review by ju-Skinner — APPROVED** [review]

**Review by QuintonJason — APPROVED** [review]

**Review by QuintonJason — APPROVED** [review]

**Pull request merged**